src/common/components/needs-settings-help-text.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedSFC } from '../react/named-sfc';
+
+export type IssueFilinigNeedsSettingsHelpTextProps = {
```

Comment:
Typo (Filinig should be Filing)
```suggestion
export type IssueFilingNeedsSettingsHelpTextProps = {
```